docs/communicate-with-javascript.md

```diff
@@ -1069,6 +1077,15 @@ type person = {
 };
 
 [@mel.module "MySchool"] external john: person = "john";
+(*
+I would explain how to create a record "manually" and latter as a bind
+let reason_person = {
+  name: "Javier",
+  friends: [| "David", "Antonio", "Jordi" |],
+  age: 99,
+};
```

Comment:
You mean, you would swap the order of the sections "Using OCaml records" and "Objects with extensible shape or unknown shape"?